models/snowflake/snowplow_fractribution_paths_to_conversion.sql

```diff
@@ -0,0 +1,32 @@
+{{ 
+  config(
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt'))
+  ) 
+}}
+
+SELECT
+  con.customerId,
+  conversionTimestamp,
+  revenue,
+  ARRAY_TO_STRING({{schema}}.TrimLongPath(
+    ARRAY_AGG(channel) WITHIN GROUP (ORDER BY visitStartTimestamp), {{ var('path_lookback_steps') }}),
+    ' > ') AS path,
+  ARRAY_TO_STRING(
+    {% for path_transform_name, _ in var('path_transforms')|reverse %}
+      {{schema}}.{{path_transform_name}}(
+    {% endfor %}
+        ARRAY_AGG(channel) WITHIN GROUP (ORDER BY visitStartTimestamp)
+    {% for _, arg_str in var('path_transforms') %}
+      {% if arg_str %}, {{arg_str}}{% endif %})
+    {% endfor %}
+    , ' > ') AS transformedPath
```

Comment:
Again for future multi-db support, could these be replaced by https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros#listagg